packages/nx/src/command-line/nx-commands.ts

```diff
@@ -618,6 +623,16 @@ function withDepGraphOptions(yargs: yargs.Argv): yargs.Argv {
         'Output file (e.g. --file=output.json or --file=dep-graph.html)',
       type: 'string',
     })
+    .option('view', {
+      describe: 'Choose whether to view the projects or task graph',
+      type: 'string',
+      default: 'projects',
+      choices: ['projects', 'tasks'],
+    })
+    .option('target', {
+      describe: 'The target to show tasks for in the task graph',
+      type: 'string',
+    })
```

Comment:
This needs to be able to accept multiple targets - for run-many or affected with multiple targets. Maybe it would be easier to pass it as something like `root-tasks`?